dashboard/app.py

```diff
@@ -0,0 +1,670 @@
+"""Python Script: Build a dashboard for data visualization"""
+import altair as alt
```

Comment:
is altair a third party library? In this case you need to change order of imports here. 
https://peps.python.org/pep-0008/#imports 
datetime would be before dotenv as well